### ISSUE TITLE
NetworkFetcher handles content-type wrongly

### ISSUE THREAD

**Issue by teejoe** [airbnb/lottie-android#1288]
http content-type header may contains charset:

> Content-Type: application/zip; charset=binary

> etc: [http://fp-dev.webapp.163.com/easybuddy/file/5d02027b54b2d310645393fa2jM9aAxj02](http://fp-dev.webapp.163.com/easybuddy/file/5d02027b54b2d310645393fa2jM9aAxj02)

but in the code only full match of "application/zip" leads to zip file composition.

NetworkFetcher.java:
```
    switch (connection.getContentType()) {
      case "application/zip":
        L.debug("Handling zip response.");
        extension = FileExtension.ZIP;
        file = networkCache.writeTempCacheFile(connection.getInputStream(), extension);
        result = LottieCompositionFactory.fromZipStreamSync(new ZipInputStream(new FileInputStream(file)), url);
        break;
      case "application/json":
      default:
        L.debug("Received json response.");
        extension = FileExtension.JSON;
        file = networkCache.writeTempCacheFile(connection.getInputStream(), extension);
        result = LottieCompositionFactory.fromJsonInputStreamSync(new FileInputStream(new File(file.getAbsolutePath())), url);
        break;
    }
```

**Comment by gpeal** [comment]
I see that your content type adds `charset=binary`. I can make this a partial match check in the next version.

**Comment by chenjianping99** [comment]
can pick this fix to version 2.7.0（not androidX version）？ because our project cannot  migrate to androidX recently
